src/OrderBook.tsx

```diff
@@ -120,7 +120,12 @@ const renderList = (
         };
 
         return (
-          <li className={`${stylePrefix}__list-item`} key={price} style={rowStyle}>
+          <li
+            data-testid={`${stylePrefix}__list-item`}
```

Comment:
I worry about adding this test ID, because now it'll show up in the DOM for anyone that uses this package.

Is there a way to write assertions that don't rely on a `testid`?

https://testing-playground.com/ could be a helpful tool to see if we can figure out a way to select this element without a `testid`. 

Taking a look in the dev tools, I suspect that you might be able to use the `listitem` role and `getByRole` to select this item. Here's a screenshot:

![image](https://user-images.githubusercontent.com/63244584/114105716-4468be00-989b-11eb-896e-f2973cb61af5.png)